SocialCareCaseViewerApi/V1/Gateways/DatabaseGateway.cs

```diff
@@ -234,9 +236,12 @@ public string GetNCReferenceByPersonId(string personId)
             return lookup?.NCId;
         }
 
-        public List<Worker> GetWorkers(int teamId)
+        public List<Worker> GetWorkers(int? teamId = null, int? id = null)
         {
-            return _databaseContext.Workers.Where(x => x.TeamId == teamId).ToList();
+            return (teamId != null) ?
```

Comment:
This statement will never return individual worker, so please review the logic after tweaking the input parameter setup